app/views/steps/case/case_type/edit.html.erb

```diff
@@ -20,7 +20,9 @@
             <%= f.govuk_radio_button :case_type, choice.value do %>
               <%= f.govuk_text_field :appeal_with_changes_maat_id, autocomplete: 'off', width: 'one-third',
                                      extra_letter_spacing: true %>
-              <%= f.govuk_text_area :appeal_with_changes_details %>
+              <%= f.govuk_text_area :appeal_with_changes_details,
+                                    'aria-live': 'polite',
+                                    'aria-label': t('helpers.label.steps_case_case_type_form.appeal_with_changes_details_a11y') %>
```

Comment:
We can use this, that way if the form object class changes name or module, this still will find the correct locale, without changes to the view:

`t("helpers.label.#{f.object_name}.appeal_with_changes_details_a11y")`

Give it a go and let me know if it doesn't work, I think I've done this in a few other places top of my head.